app/scripts/modules/injected/message.js

```diff
@@ -1,13 +1,56 @@
 'use strict';
 
+/**
+ * Parses given object into a JSON object removing all functions and handle circular references.
+ * @param {Object} object input object
+ * @returns {Object} JSON object
+ * @private
+ */
+function _prepareMessage(object) {
+    if (object === null || object === undefined || typeof object === "function") {
```

Comment:
Since this is an internal module that we create and it is used with internal purpose I think we can remove this checks for the moment

Reply:
I am not a fan of having any input unchecked in JS, even for privat methods. However, jshint CC check also prefers less complexity 😮